### PR TITLE
Asistencia 26 y challenges

### PR DESCRIPTION
**Pull request by LawlietLelouch03**

### CODE REVIEW

File: VicenteDouriet/ChallengesDay26/Hackerrank/SolutionInheritance2.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  private int b;
+  
+  void add(int a,int b) {
+    System.out.print(a+b+" ");
+    
+  }
+}
+class Adder extends Arithmetic {
+    Adder(){ 
+      System.out.println("My superclass is: Arithmetic");
```
Comment (edbermeo) [inline VicenteDouriet/ChallengesDay26/Hackerrank/SolutionInheritance2.java:13]: this is hardcoded, what if I change the Arithmetic class name?
check 
getClass().getSimpleName()

File: VicenteDouriet/ChallengesDay26/Hackerrank/SolutionInheritance2.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class Arithmetic
```
Comment (edbermeo) [inline VicenteDouriet/ChallengesDay26/Hackerrank/SolutionInheritance2.java:1]: Use a file per class

File: VicenteDouriet/ChallengesDay26/Hackerrank/SolutionInheritance1.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+class Animal {
```
Comment (edbermeo) [inline VicenteDouriet/ChallengesDay26/Hackerrank/SolutionInheritance1.java:1]: Use a file per class

File: VicenteDouriet/ChallengesDay26/ChallengesClass/Employee.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    }
+    Employee(String name, int age, double salary){
+        this.name = name;
+        this.age = age;
+        this.salary = salary;
+    }
+
+
+
+    public void printData(){
```
Comment (edbermeo) [inline VicenteDouriet/ChallengesDay26/ChallengesClass/Employee.java:17]: this can be extracted to an interface

**Review by edbermeo — CHANGES_REQUESTED** [review]
check code formatting and indentation

**Comment by edbermeo** [comment]
Please attend comments

**Review by luisalanmz — APPROVED** [review]
LGTM

File: VicenteDouriet/ChallengesDay26/Hackerrank/SolutionInheritance1/Bird.java [hunk 5]
```diff
@@ -0,0 +1,8 @@
+class Bird extends Animal {
```
Comment (luisod78) [inline VicenteDouriet/ChallengesDay26/Hackerrank/SolutionInheritance1/Bird.java:1]: the default unnamed package should not be used

File: VicenteDouriet/ChallengesDay26/ChallengesClass/Tester.java [hunk 6]
```diff
@@ -0,0 +1,10 @@
+public class Tester extends Employee {
```
Comment (luisod78) [inline VicenteDouriet/ChallengesDay26/ChallengesClass/Tester.java:1]: the default unnamed package should not be used

File: VicenteDouriet/ChallengesDay26/ChallengesClass/Printable.java [hunk 7]
```diff
@@ -0,0 +1,4 @@
+public interface Printable {
```
Comment (luisod78) [inline VicenteDouriet/ChallengesDay26/ChallengesClass/Printable.java:1]: the default unnamed package should not be used

File: VicenteDouriet/ChallengesDay26/ChallengesClass/Developer.java [hunk 8]
```diff
@@ -0,0 +1,8 @@
+public class Developer extends Employee {
```
Comment (luisod78) [inline VicenteDouriet/ChallengesDay26/ChallengesClass/Developer.java:1]: the default unnamed package should not be used

File: VicenteDouriet/ChallengesDay26/ChallengesClass/Company.java [hunk 9]
```diff
@@ -0,0 +1,27 @@
+public class Company {
```
Comment (luisod78) [inline VicenteDouriet/ChallengesDay26/ChallengesClass/Company.java:1]: the default unnamed package should not be used

File: VicenteDouriet/ChallengesDay26/Hackerrank/SolutionPrintf/SolutionPrintf.java [hunk 10]
```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+public class SolutionPrintf {
+
+   public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String equalsInALine = "================================"
+        System.out.println(equalsInALine);
+        for (int i = 0; i < 3; i++) {
```
Comment (luisod78) [inline VicenteDouriet/ChallengesDay26/Hackerrank/SolutionPrintf/SolutionPrintf.java:9]: Variables names must be meaningful 

Avoid using magic numbers and magic strings

**Pull request closed without merging**